src/components/MdComponents.tsx

```diff
@@ -32,44 +34,73 @@ import InfoBanner from "./InfoBanner"
 import Card from "./Card"
 import QuizWidget from "./Quiz/QuizWidget"
 import DocLink from "./DocLink"
+import Link from "@/components/Link"
+import { motion } from "framer-motion"
```

Comment:
```suggestion
```
I don't see this used anywhere in this component

Reply:
Ah I realize this PR was made off of https://github.com/ethereum/ethereum-org-fork/pull/49/files, commented in this PR about this.